sample.html

```diff
@@ -2242,6 +2276,13 @@ <h4 class="panel-title">Options</h4>
         $("#pxlSizeActive").prop('checked', false);
         $("#pxlSizeGroup").css('display', 'none');
 
+        $("#pxlBoundX").val('');
+        $("#pxlBoundY").val('');
```

Comment:
I think these values should be set to 0.

If these values are null when the `config` is sent, you'll get this error.

```
[ERROR] 2020-01-30 19:04:51,758 @ qz.utils.PrintingUtilities:?
	Failed to print
java.lang.NumberFormatException: Character N is neither a decimal digit number, decimal point, nor "e" notation 
```

Reply:
> I think these values should be set to 0.

We do this for `margins: {}`, but not for `size: {}`.  We can default `x: 0` and `y: 0`, but for consistency in the sample, I don't think we should default the width and height.

Reply:
Ok, turns out `optDouble` strangely divides by zero when the value isn't parsable.  Patched via 
1e9af8b, a95bf4b.